roles/confluent.ssl/tasks/self_signed_certs.yml

```diff
@@ -74,19 +82,39 @@
 
 - name: Import the CA cert into the Keystore
   shell: |
-    keytool -noprompt -keystore {{keystore_path}} \
+    keytool -noprompt -keystore /var/ssl/private/{{service_name}}.keystore.jks \
       -storetype pkcs12 \
       -alias CARoot \
       -import -file {{ca_cert_path}} \
       -storepass {{keystore_storepass}} \
-      -keypass {{keystore_storepass}} {{extra_args}}
+      -keypass {{keystore_storepass}}
+
+- name: Create Cert Chain
+  shell: |
+    cat {{ca_cert_path}} {{cert_path}} > /var/ssl/private/generation/signed-cert-chain.crt
 
 - name: Import the Signed Host Cert into the Keystore
   shell: |
-    keytool -noprompt -keystore {{keystore_path}} \
+    keytool -noprompt -keystore /var/ssl/private/{{service_name}}.keystore.jks \
       -storetype pkcs12 \
       -alias {{inventory_hostname}} \
-      -import -file {{cert_path}} \
+      -import -file /var/ssl/private/generation/signed-cert-chain.crt \
       -ext "SAN={{hostnames | cert_extension}}" \
       -storepass {{keystore_storepass}} \
-      -keypass {{keystore_storepass}} {{extra_args}}
+      -keypass {{keystore_storepass}}
+
```

Comment:
Missing logic for custom certs option, also investigate provided keystore/truststore